### ISSUE TITLE
CCDC160 should not be part of PTHR11636

### ISSUE THREAD

**Issue by pgaudet** [pantherdb/Helpdesk#23]
CCDC160 (A6NGH7) is has low sequence similarity with other members of this family, and does not contain a HOX domain, it should be removed from this family. 

Thanks, Pascale

**Comment by dustine32** [comment]
Thanks @pgaudet!

Gonna throw this to @huaiyumi. If, in the next version (16.0), we removed the PTHR11636 family assignment for A6NGH7 and depended on HMM alignment to find a new family then it would still go into PTHR11636. I confirmed this with a PANTHER sequence search (using [this sequence](https://www.uniprot.org/uniprot/A6NGH7.fasta)):
![image](https://user-images.githubusercontent.com/2678599/90932578-daef7a00-e3b3-11ea-9c94-faccb9a4a27b.png)

So do we just want to completely remove this sequence for PANTHER 16.0 or manually find the correct family?
